src/assets/scripts/client/game/GameController.js

```diff
@@ -181,27 +231,45 @@ class GameController {
 
         // Reset score
         this.game.score = 0;
+
+        this.game_updateScore();
+    }
+
+    /**
+     *
+     * @for GameController
+     * @method updateTimewarp
+     * @param nextValue {number}
+     */
+    updateTimewarp(nextValue) {
+        if (nextValue === 0) {
+            this.game_timewarp_toggle();
+
+            return;
+        }
+
+        TimeKeeper.updateTimewarp(nextValue);
     }
 
     /**
      * @for GameController
      * @method game_timewarp_toggle
      */
     game_timewarp_toggle() {
-        const $fastForwards = $(`.${SELECTORS.CLASSNAMES.FAST_FORWARDS}`);
+        const $fastForwards = $(SELECTORS.DOM_SELECTORS.FAST_FORWARDS);
 
-        if (this.game.speedup === 5) {
-            this.game.speedup = 1;
+        if (TimeKeeper.timewarp === 5) {
```

Comment:
We should make this `if (TimeKeeper.timewarp >= 5) {` so that custom timewarps show up as red toggle, and can be reset to 1 by pressing the toggle once.

Reply:
done